conf/core/ConfigOptions.asciidoc

```diff
@@ -1567,20 +1568,24 @@ Not necessarily utilized by all parts of the application.
 ** `hoot::DuplicateNameRemover`
 ** `hoot::SmallWayMerger`
 ** `hoot::RemoveEmptyAreasVisitor`
+** `hoot::RemoveDuplicateRelationMembersVisitor`
+** `hoot::RelationCircularRefRemover`
+** `hoot::RemoveEmptyRelationsOp`
 ** `hoot::RemoveDuplicateAreaVisitor`
```

Comment:
Added two new ops and an existing one to cleaning.  The existing one was running post conflate but not during cleaning yet.